controllers/cinder_controller.go

```diff
@@ -920,8 +872,20 @@ func (r *CinderReconciler) generateServiceConfigs(
 
 	labels := labels.GetLabels(instance, labels.GetGroupLabel(cinder.ServiceName), serviceLabels)
 
+	db, err := mariadbv1.GetDatabaseByName(ctx, h, cinder.DatabaseName)
+	if err != nil {
+		return err
+	}
+
+	var tlsCfg *tls.Service
+	if instance.Spec.CinderAPI.TLS.Ca.CaBundleSecretName != "" {
+		tlsCfg = &tls.Service{}
+	}
 	// customData hold any customization for all cinder services.
-	customData := map[string]string{cinder.CustomConfigFileName: instance.Spec.CustomServiceConfig}
+	customData := map[string]string{
+		cinder.CustomConfigFileName: instance.Spec.CustomServiceConfig,
+		"my.cnf":                    db.GetDatabaseClientConfig(tlsCfg), //(mschuppert) for now just get the default my.cnf
```

Comment:
Also, I prefer to avoid hard-coding the same string in multiple locations. Consider defining (and using) this in pkg/cinder/const.go along with the other XxxFileName constants:
```
MyCnfFileName = "my.cnf"
```

Reply:
ok, I can do that

Reply:
done